.github/workflows/post-merge.yml

```diff
@@ -0,0 +1,63 @@
+name: Post Merge
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/post-merge.yml
```

Comment:
This can be removed once we're happy with this PR.

Reply:
LGTM - do you want to remove and I'll approve?

Reply:
Removed